### PR TITLE
NEXUS-10209-License_Improvements

### PR DESCRIPTION
**Pull request by joedragons**
* Added a line regarding the added warning.  Rest of the text seems to already match what we do (did).
* Fixed a typo (in the area).
* Removed an extra whitespace (in the area).
* Removed an extra whitespace (eof).

https://issues.sonatype.org/browse/NEXUS-10209

### CODE REVIEW

File: chapter-configuration.asciidoc [hunk 1]
```diff
@@ -918,8 +918,7 @@ agent'). To generate the report click 'Download' to produce a CSV file of listed
 
 When a {pro} license expires all functionality will be disabled in the user interface, except for the ability to 
 install a new license file. To avoid interruption of service be sure to upload a renewed license before the 
-existing license expi
-
+existing license expires.  {pro} will provide a warning when the license in within 30 days of expiry.
```
Comment (dbradicich) [inline chapter-configuration.asciidoc]: license ```IS``` within 30 days

**Comment by dbradicich** [comment]
+1

**Comment by fjmilens3** [comment]
+1

File: chapter-configuration.asciidoc [hunk 2]
```diff
@@ -918,8 +918,7 @@ agent'). To generate the report click 'Download' to produce a CSV file of listed
 
 When a {pro} license expires all functionality will be disabled in the user interface, except for the ability to 
 install a new license file. To avoid interruption of service be sure to upload a renewed license before the 
-existing license expi
-
+existing license expires.  {pro} will provide a warning when the license is within 30 days of expiry.
```
Comment (mosabua) [inline chapter-configuration.asciidoc]: Do not use future tense ... {pro} provides...

**Comment by joedragons** [comment]
@mosabua I will endeavor to follow this rule - for this example, I followed the existing text and even if I had probably wouldn't have rewritten the whole thing as it seems out of scope (I've been previously told bigger edits make the PR harder to review...such as 114izing).

File: chapter-configuration.asciidoc (hunk 2 above)
Comment (joedragons) [inline chapter-configuration.asciidoc]: @mosabua I will endeavor to follow this rule - for this example, I followed the existing text and even if I had probably wouldn't have rewritten the whole thing as it seems out of scope (I've been previously told bigger edits make the PR harder to review...such as 114izing).

**Pull request merged**